modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryTableStorage.java

```diff
@@ -164,7 +164,7 @@ private FreeListImpl createFreeList(
             boolean initNew = false;
 
             if (meta.freeListRootPageId() == 0) {
-                long rootPageId = pageMemory.allocatePage(getTableId(), partId, FLAG_AUX);
+                long rootPageId = pageMemory.allocatePageNoReuse(getTableId(), partId, FLAG_AUX);
```

Comment:
Why no reuse here?

Reply:
We use free lists to reuse pages, so when creating free list itself we have nothing to reuse pages from. Or I'm wrong?

Reply:
So starting persistent partiton without previous meta we actually don't reuse any pages. Free list is just created, it is empty and we have to allocate new pages for all structures. Don't know what to do with that

Reply:
We have both a `FreeList` and a `ReuseList`, I always confuse them. I thought we used `ReuseList` for empty pages